spec/rubocop/cop/lint/non_atomic_file_operation_spec.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Lint::NonAtomicFileOperation, :config do
+  shared_examples 'with implicit receiver' do |method_id|
```

Comment:
Probably, `method_name` would be more appropriate.

```suggestion
  shared_examples 'with implicit receiver' do |method_name|
```